src/content/en/tools/lighthouse/audits/passive-event-listeners.md

```diff
@@ -0,0 +1,62 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Site Uses Passive Event Listeners to Improve Scrolling Performance" Lighthouse audit.
+
+{# wf_updated_on: 2016-11-29 #}
+{# wf_published_on: 2016-11-29 #}
+
+# Site Uses Passive Event Listeners to Improve Scrolling Performance  {: .page-title }
+
+## Why the audit is important {: #why }
+
+Setting the `passive` option on your wheel and touch event listeners can
```

Comment:
I'm using these as general categories. `wheel` is one of the actual event names, but `touch` is not, so it'd be confusing to write `touch` because no such event exists